tests/test_manage.py

```diff
@@ -4,27 +4,40 @@
 In this case, we are testing the manage.py script, so we need to check
 if it can be imported and if it runs correctly.
 """
-import pytest
 import subprocess
-
 from unittest.mock import patch
 
+import pytest
+
 
 def test_manage():
+    """
+    Test the importability of the main function from manage.py.
+
+    This test checks if the main function from manage.py can be imported
+    without errors, indicating the script is structured correctly.
+    """
     try:
-        # Intenta importar la función principal de manage.py
+        # try importing the main function from manage.py
         from manage import main
         assert main is not None
     except Exception as e:
         pytest.fail(f"manage.py import failed: {e}")
 
 
 def test_manage_raise_exception():
-    # use 'execute_from_command_line' and not 'import_module', becasue 'import' preloads the module and we want to test!
+    """
+    Test manage.py for proper error handling when Django is not importable.
+
+    This test mocks a failed Django import to ensure that manage.py
+    handles such exceptions as expected.
+    """
+    # use 'execute_from_command_line' and not 'import_module',
+    # becasue 'import' preloads the module and we want to test!
     def mock_execute_from_command_line(*args, **kwargs):
         raise ImportError("Couldn't import Django")
 
-    with patch("django.core.management.execute_from_command_line", mock_execute_from_command_line):
+    with patch("django.core.management.execute_from_command_line", mock_execute_from_command_line):  # noqa: E501
```

Comment:
🤨

Reply:
in this case, only for exceptions, can you skip the review for this rule:)
I take this decision because I dont like 
<break>
the strings:)